packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2264,9 +2265,20 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     /**
      * When running GC in test mode, this is called to delete objects whose routes are unused. This enables testing
      * scenarios with accessing deleted content.
-     * @param unusedRoutes - The routes that are unused in all data stores in this Container.
+     * @param unusedRoutes - The routes that are unused in all data stores and blobs in this Container.
      */
     public deleteUnusedRoutes(unusedRoutes: string[]) {
+        /**
+         * When running GC in tombstone mode, this is called to tombstone datastore routes that are unused. This
+         * enables testing scenarios without actually deleting content. The content acts as if it's deleted to the
+         * external user, but the internal runtime does not delete it in summarizes, etc.
+         */
+        const tombstone = this.mc.config.getBoolean(testTombstoneKey) ?? false;
```

Comment:
GarbageCollector is already checking this flag. It can simply pass in a tombstone param to deleteUnusedRoutes here and container runtime can be a pass through. Also, ideally only GarbageCollector should look at GC related flags.

Reply:
The only reason why I didn't pass the param was because it would make it a breaking change as it technically changes the API layer.

Reply:
I see. It's not a big deal. You can leave it this way and we can clean this up later. I would like to remove the export of IGarbageCollectionRuntime so that we can do this kind of changes without worrying about back-compat.

Reply:
I too would really like doing this as well. This is all internal runtime anyway